tests/unit/testplan/testing/multitest/test_result.py

```diff
@@ -302,3 +303,55 @@ def test_report_modes(self, fix_ns):
         assert len(fix_ns.result.entries) == 1
         dict_assert = fix_ns.result.entries.popleft()
         assert len(dict_assert.comparison) == 1
+
+    def test_graph_assertion(self):
+        """Unit testcases for the result.graph class."""
+        result = result_mod.Result()
+        graph_assertion = result.graph(
+                                        'Line',
+                                        {'Data Name':  [
+                                                     {'x': 0, 'y': 8},
+                                                     {'x': 1, 'y': 5},
+                                                     {'x': 2, 'y': 4},
+                                                     {'x': 3, 'y': 9},
+                                                     {'x': 4, 'y': 1},
+                                                     {'x': 5, 'y': 7},
+                                                     {'x': 6, 'y': 6},
+                                                     {'x': 7, 'y': 3},
+                                                     {'x': 8, 'y': 2},
+                                                     {'x': 9, 'y': 0}
+                                                        ]
+                                        },
+                                        description='Line Graph',
+                                        series_options={
+                                                         'Data Name': {"colour": "red"}
+                                                    },
+                                        graph_options=None
+                                    )
+        assert graph_assertion is True
+
```

Comment:
After making a graph assertion, you'll want to check that the graph data has been appended to the `result.entries` list. Assert that the data is stored as you expect.